lambda/update/validate.go

```diff
@@ -16,8 +16,8 @@ func validateUpdate(update shared.Update, lpa *shared.Lpa) (Applyable, []shared.
 		return validateCertificateProviderOptOut(update.Changes)
 	case "CERTIFICATE_PROVIDER_SIGN":
 		return validateCertificateProviderSign(update.Changes, lpa)
-	case "PERFECT":
-		return validatePerfect(update.Changes)
+	case "STATUTORY_WAITING_PERIOD":
+		return validateStatutoryWaitingPeriod(update.Changes)
```

Comment:
I think we need to keep `PERFECT` to continue supporting MRLPA, but add a new name alongside to allow us to expand-contract towards it.

For consistency it would be better to use a verb than a noun, so maybe `START_STATUTORY_WAITING_PERIOD`?
```suggestion
	case "PERFECT":
		return validatePerfect(update.Changes)
	case "START_STATUTORY_WAITING_PERIOD":
		return validateStatutoryWaitingPeriod(update.Changes)
```

Reply:
would this work: 
```
	case "PERFECT":
	case "STATUTORY_WAITING_PERIOD":
		return validateStatutoryWaitingPeriod(update.Changes)
```
or do I need to bring back all of the perfect files too?